app/bundles/WebhookBundle/Form/Type/WebhookType.php

```diff
@@ -94,14 +79,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $builder->add(
             'events',
-            'choice',
+            ChoiceType::class,
             [
-                'choices'    => $choices,
-                'multiple'   => true,
-                'expanded'   => true,
-                'label'      => 'mautic.webhook.form.webhook.events',
-                'label_attr' => ['class' => 'control-label'],
-                'attr'       => ['class' => ''],
+                'choices'           => array_flip($choices),
```

Comment:
Flip the `$type` and label in the foreach above when the `$choices` array is being built. Then we can get rid of the array_flip function and make this more performant.